python/pyspark/sql/protobuf/functions.py

```diff
@@ -57,7 +57,7 @@ def from_protobuf(
         without descFilePath parameter.
```

Comment:
@rangadi can we also add options for from_protobuf and to_protobuf in pyspark API too?

Reply:
@SandishKumarHN this PR does that. See below. `from_protobuf()' already has. Added options for `to_protobuf()`. I will update the doc.